README.md

```diff
@@ -45,3 +45,25 @@ log as requests are made to them.
 You can test out the FIDS/Alerts sample application by visiting http://localhost:8080 in
 your web browser (if not running Docker locally, use the Docker host's
 address).
+
+## Alerts Backend Note:
+
+Please note that when you create an alert, whenever an event is triggered for that
+alert, AeroAPI will send a POST request to your configured endpoint. In order to configure
+your endpoint, the server you are running the app on will need to have a publicly exposed
+address/port (using the POST_PORT environment variable, NOT the WEB_SERVER_PORT variable).
+Furthermore, as noted in the docker-compose-alerts.yml file, we encourage the service for
```

Comment:
I think you can delete "Furthermore, as noted in the docker-compose-alerts.yml file, we encourage the service for accepting POSTed triggered alerts to be isolated, and thus will be sent to a different Docker
container. This means that you will have to set the endpoint URL using /post, instead of /api/post."